test/suites/rpm/install-and-upgrade-successful.robot

```diff
@@ -54,9 +54,9 @@ Upgrade From Previous Version
     # Depending on where we are in the current release cycle, we might
     # need to install from an unofficial repository if the previous
     # release only has EC and RC builds.
-    IF    '${PREVIOUS_VERSION_REPO_URL}' != '${EMPTY}'
+    IF    '${PREVIOUS_VERSION_REPO}' != '${EMPTY}'
         Install MicroShift RPM Packages From Repo
-        ...    ${PREVIOUS_VERSION_REPO_URL}
+        ...    ${PREVIOUS_VERSION_REPO}
         ...    4.${PREVIOUS_MINOR_VERSION}.*
     ELSE
```

Comment:
That `ELSE` branch is no longer necessary - ${PREVIOUS_VERSION_REPO} will always have a value - sometimes it's rhocp, sometimes it's openshift mirror, sometimes it's http repo from hypervisor